pkgs/by-name/taldir/package.nix

```
@@ -6,19 +6,19 @@
 }:
 buildGoModule {
```

Here, we can use `finalAttrs` to refer to attributes from within the derivation (e.g finalAttrs.version). We can also use `rec`, which will allow us to access `version` directly, but that comes with its [own issues](https://nix.dev/guides/best-practices#recursive-attribute-set-rec), so it's best to use `finalAttrs` when we can.

```suggestion
buildGoModule (finalAttrs: {
```

Note: `finalAttrs` is also better for overriding the package, but maybe that's a topic for another time 😄

---

Thank you for the feedback. The issues and your feedbacks are helping me learn more about nix as I go 😅